app/controllers/ITSAStatusController.scala

```diff
@@ -43,6 +43,9 @@ class ITSAStatusController @Inject()(authentication: AuthenticationPredicate,
       case Left(error: ITSAStatusResponseError) =>
         logger.error(s"[ITSAStatusController][getITSAStatus] - Error Response: $error")
         Status(error.status)(Json.toJson(error))
+      case Left(_) =>
```

Comment:
can you pls provide error details in the log ??

Reply:
I mean something like:
```
      case Left(error) =>
        logger.error(s"[ITSAStatusController][getITSAStatus][ITSAStatusResponseModel] - Error: $error")
        InternalServerError(s"[ITSAStatusController][getITSAStatus]")
```

Reply:
case Left(error) statement would capture any type of error and should be fine to use